app/src/main/java/com/iic/shopingo/ui/request_flow/activities/CreateShoppingListActivity.java

```diff
@@ -49,8 +58,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
   @OnClick(R.id.create_request_create_button)
   public void onCreateRequest(View view) {
-    request.items = itemListView.getAllItems();
-    request.price = Integer.parseInt(priceView.getText().toString());
+    shoppingList.setItems(itemListView.getAllItems());
+    shoppingList.setOffer(Integer.parseInt(priceView.getText().toString()));
+    request.setShoppingList(shoppingList);
+    request.setStatus(BaseRequest.RequestStatus.PENDING);
```

Comment:
It just makes everything much easier to pass around, especially after canceling a request or getting a decline, you can just pass around the request which already contains everything.